server/config/config_test.go

```diff
@@ -22,6 +22,7 @@ func TestIsValid(t *testing.T) {
 				AzureDevopsOAuthAppID:        "mockAzureDevopsOAuthAppID",
 				AzureDevopsOAuthClientSecret: "mockAzureDevopsOAuthClientSecret",
 				EncryptionSecret:             "mockEncryptionSecret",
+				WebhookSecret:                "mockWebhookSecret",
 			},
 			errMsg: "",
```

Comment:
```suggestion
```

Reply:
giving error on removing it @raghavaggarwal2308 can look into it

Reply:
@avas27JTG Fixed [here](https://github.com/Brightscout/mattermost-plugin-azure-devops/pull/22#pullrequestreview-1243581764)